diff --git a/src/include/gsh_rpc.h b/src/include/gsh_rpc.h
index 405b1c339..bee860198 100644
--- a/src/include/gsh_rpc.h
+++ b/src/include/gsh_rpc.h
@@ -46,23 +46,23 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-#include <rpc/xdr_inline.h>
+#include <ntirpc/rpc/xdr_inline.h>
 #ifdef __cplusplus
 }
 #endif
 
-#include <rpc/rpc.h>
-#include <rpc/svc.h>
-#include <rpc/clnt.h>
+#include <ntirpc/rpc/rpc.h>
+#include <ntirpc/rpc/svc.h>
+#include <ntirpc/rpc/clnt.h>
 #include <arpa/inet.h>
 
-#include <rpc/svc_auth.h>
+#include <ntirpc/rpc/svc_auth.h>
 #ifdef _HAVE_GSSAPI
-#include <rpc/auth_gss.h>
-#include <rpc/gss_internal.h>	/* XXX */
+#include <ntirpc/rpc/auth_gss.h>
+#include <ntirpc/rpc/gss_internal.h>	/* XXX */
 #endif
-#include <rpc/svc_rqst.h>
-#include <rpc/rpc_msg.h>
+#include <ntirpc/rpc/svc_rqst.h>
+#include <ntirpc/rpc/rpc_msg.h>
 
 #include "common_utils.h"
 #include "abstract_mem.h"
Submodule src/libntirpc contains modified content
