diff --git a/cmake/Libraries.cmake b/cmake/Libraries.cmake
index 2cb478d6..db68a17c 100644
--- a/cmake/Libraries.cmake
+++ b/cmake/Libraries.cmake
@@ -174,9 +174,11 @@ message(STATUS "ISAL PIC LIBRARY: ${ISAL_PIC_LIBRARY}")
 # Download nfs-ganesha
 
 if(ENABLE_NFS_GANESHA)
-  download_external(NFS_GANESHA "nfs-ganesha-4.3"
-                    "https://github.com/nfs-ganesha/nfs-ganesha/archive/V4.3.zip")
-  download_external(NTIRPC "ntirpc-4.3"
-                    "https://github.com/nfs-ganesha/ntirpc/archive/v4.3.zip")
+  find_library(NFS_GANESHA nfs-ganesha)
+  find_library(NTIRPC ntirpc)
+  # download_external(NFS_GANESHA "nfs-ganesha-4.3"
+  #                   "https://github.com/nfs-ganesha/nfs-ganesha/archive/V4.3.zip")
+  # download_external(NTIRPC "ntirpc-4.3"
+  #                   "https://github.com/nfs-ganesha/ntirpc/archive/v4.3.zip")
 endif()
 
